obspy/io/nlloc/tests/data/nlloc_custom.qml

```diff
@@ -157,63 +157,63 @@ STATISTICS ExpectX 4473.68 Y 5323.29 Z 4.59501  CovXX 0.0282621 XY -0.0053866 XZ
           <value>2010-05-27T16:56:25.930000Z</value>
           <uncertainty>0.02</uncertainty>
         </time>
-        <waveformID networkCode="" stationCode="UH3"></waveformID>
+        <waveformID channelCode="?" stationCode="UH3"></waveformID>
         <phaseHint>P</phaseHint>
       </pick>
       <pick publicID="smi:local/3a0bde89-d7e6-45ef-a08a-4b950720f7be">
         <time>
           <value>2010-05-27T16:56:27.100000Z</value>
           <uncertainty>0.06</uncertainty>
         </time>
-        <waveformID networkCode="" stationCode="UH3"></waveformID>
+        <waveformID channelCode="?" stationCode="UH3"></waveformID>
```

Comment:
channel code `"?"`?

Reply:
Yea, these where in [one of the nlloc hyp files](https://github.com/obspy/obspy/blob/master/obspy/io/nlloc/tests/data/nlloc_custom.hyp#L17). I don't know. What do you suggest? Maybe do not set the channel when it is `?`. What with channel codes like `HH?`? Additionally raise a warning?